### PR TITLE
Basic streaming support for postgresql LOB storage

### PR DESCRIPTION
**Pull request by wedens**

### CODE REVIEW

File: yax/postgres/src/main/scala/doobie/postgres/hi/largeobjectmanager.scala [hunk 1]
```diff
@@ -38,4 +38,11 @@ object largeobjectmanager {
   def createFileFromLO(blockSize: Int, oid: Long, file: File): LargeObjectManagerIO[Unit] =
     open(oid)(PHLO.copyToFile(blockSize, file))
 
+  def createLOFromStream(blockSize: Int, is: InputStream): LargeObjectManagerIO[Long] =
+    PHLOM.createLO >>= { oid =>
+      PHLOM.open(oid)(PHLO.copyFromStream(blockSize, is)).as(oid)
+    }
+
+  def createFileFromLO(blockSize: Int, oid: Long, os: OutputStream): LargeObjectManagerIO[Unit] =
```
Comment (tpolecat) [inline yax/postgres/src/main/scala/doobie/postgres/hi/largeobjectmanager.scala]: `File` doesn't appear in the types here ... maybe `copyToStreamFromLO`?

Comment (wedens) [inline yax/postgres/src/main/scala/doobie/postgres/hi/largeobjectmanager.scala]: sure. forgot to rename

**Comment by wedens** [comment]
done

**Comment by tpolecat** [comment]
Thanks!

**Pull request merged**